### DISCUSSION TITLE
CERTIFICATE_VERIFY_FAILED error

### DISCUSSION THREAD

**Discussion by gulnuravci** [mrdbourke/pytorch-deep-learning#695] [Q&A]
I came across this issue while doing _06. PyTorch Transfer Learning_ section, following these steps fixed the issue for me: 
[https://stackoverflow.com/questions/68275857/urllib-error-urlerror-urlopen-error-ssl-certificate-verify-failed-certifica](https://stackoverflow.com/questions/68275857/urllib-error-urlerror-urlopen-error-ssl-certificate-verify-failed-certifica)

**Comment by mrdbourke** [comment]
Hey @gulnuravci , 

Glad to hear you found a solution to your error!

I'm curious to know, where did it come from?

**Comment by gulnuravci** [comment]
Hi @mrdbourke,

It first happened when I tried running `model = efficientnet_b0(weights=weights)` in the transfer learning section of the course. Once I followed the steps in the link I included I was able to import the model without errors.
